internal/pkg/cli/pipeline_init.go

```diff
@@ -566,16 +495,19 @@ func (o *initPipelineOpts) selectBranch() error {
 // bbssh	ssh://git@bitbucket.org:teamsinspace/documentation-tests.git (fetch)
 
 // parseGitRemoteResults returns just the first (remote name) and second (url) columns of the `git remote -v` results as a map (url: name), and skips urls from unsupported sources.
-func (o *initPipelineOpts) parseGitRemoteResult(s string) map[string]string {
-	repos := make(map[string]string)
+func (o *initPipelineOpts) parseGitRemoteResult(s string) []repoResult {
```

Comment:
hmm why do we use `[]repoResult` instead of a map?

Reply:
I changed it to a struct bc it felt more in line with the rest of our code, and easier to format then parse.